File: src/main/java/codereview/simpleorder/order/dto/OrderLineRequest.java
package codereview.simpleorder.order.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@NoArgsConstructor
@AllArgsConstructor
@Getter
public class OrderLineRequest {

    private Long itemId;
    private int quantity;
}

File: order/src/main/kotlin/io/philo/shop/dto/web/OrderLineRequest.kt
package io.philo.shop.dto.web

import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor

@NoArgsConstructor
@AllArgsConstructor
@Getter
data class OrderLineRequest(
    val itemId: Long,
    val quantity: Int = 0
)